openedx/features/qverse_features/registration/forms.py

```diff
@@ -3,16 +3,20 @@
 """
 from django.forms import ModelForm
 
-from openedx.features.qverse_features.registration.models import QVerseUserProfile
+from openedx.features.qverse_features.registration.models import QVerseUserProfile, ProspectiveUser
+from student.helpers import AccountValidationError
 
 
 class QVerseUserProfileForm(ModelForm):
     """
     Form to add custom registration fields.
     """
+    def __init__(self, *args, **kwargs):
```

Comment:
If we are not doing anything here, can we omit this ?

Reply:
done.